canonical-web-and-design/ubuntu.com
# [Certified] - Build vendor pages view and template

🆘 First Python PR!! Please help 🆘 

## Done

- Added some information about the vendors to the `vendors_data.yaml` file
- Built template at `/certified/vendors/vendor.html`
- Added end point at `/certified/vendors/<vendor>`
- Added a new view function `certified_vendors`. 

## Still to do

- The filters on the left should only show which ones are applicable for example if there are no servers by dell then the filter shouldn't exist. Not sure how to implement this. I've spoken to Sheila and getting categories/releases by vendor isn't something the API provides.

## QA

- View page at: https://ubuntu-com-10692.demos.haus
- Go to `/certified/laptops` and click the dell link in the vendor column of the table. Check it takes you to the correct dell specific search results page
- This can be repeated for all the vendors in which we have data for in the yaml file: 
  - Dell
  - Hp
  - Lenovo
  - Intel
  - Xilinx
  - Huawei
  - Cisco
  - Ericsson
  - IBM
  -  Mellanox Technologies
  - NEC
  - Rigado
  - Nvidia
- On this page check the filters and search bar work as expected
- Go to `/certified/devices` and search for advantech
- In the search results check the link to Advantech link takes you to the generic search results page with Advantech results, as this vendor doesn't have it's own specific page. 


## Issue / Card

Fixes [#4261](https://github.com/canonical-web-and-design/web-squad/issues/4261)
